<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FiveChessForm</class>
 <widget class="QWidget" name="FiveChessForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game  Widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>962</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>return</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QPushButton" name="helpButton">
        <property name="text">
         <string>help</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="blackLable">
        <property name="text">
         <string>Black player</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="blackRecordLabelPre">
        <property name="text">
         <string>win</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="blackRecordWinLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="blackRecordLabelMiddle">
        <property name="text">
         <string>times,lose</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="blackRecordLoseLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="blackRecordLabelAfter">
        <property name="text">
         <string>times</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QPushButton" name="blackGiveupButton">
        <property name="text">
         <string>giveup</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QPushButton" name="restoreButton">
        <property name="text">
         <string>retrack</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>save game</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="whiteLabel">
        <property name="text">
         <string>White player</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="whiteRecordLabelPre">
        <property name="text">
         <string>win</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="whiteRecordWinLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="whiteRecordLabelMiddle">
        <property name="text">
         <string>times,lose</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="whiteRecordLoseLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="whiteRecordLabelAfter">
        <property name="text">
         <string>times</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QPushButton" name="whiteGiveupButton">
        <property name="text">
         <string>giveup</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
